src/Sylius/Bundle/ApiBundle/composer.json

```diff
@@ -20,18 +20,19 @@
         }
     ],
     "require": {
-        "php":                      ">=5.5.9",
+        "php":                                  ">=5.5.9",
 
+        "friendsofsymfony/oauth-server-bundle": "~1.4",
         "sylius/resource-bundle":               "0.16.*",
-        "symfony/framework-bundle":             "~2.3",
-        "friendsofsymfony/oauth-server-bundle": "~1.4"
+        "symfony/framework-bundle":             "~2.3"
```

Comment:
https://github.com/Sylius/Sylius/pull/3449/files update to symfony 2.7.5 has not been merged yet :)